cmd/query/app/testdata/example-CA-cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is it necessary to create a new set of files? can we just update the already existing files and load them all from a single location?

Reply:
Sure.  That would be great.  I'll do that as a separate PR perhaps.  I ran a bash command to find all the .pem files in the project.  That can be found here:  https://pastebin.ubuntu.com/p/gJ5DmtYZsF/   .  It would be great if you could tell me the ones that could be replaced by the files from the central location (which i think is `./pkg/config/tlscfg/testdata`).  I am not well versed on the purpose of each one.   

1.  Please tell me which files are to be replaced by central certificates (As of now,  I am sure that reporter, query and tlscfg 's testdata can be replaced.)
1.  Please also tell me the path/location where i need to store the keys (   `./pkg/config/tlscfg/testdata`  makes it seem like it is sepcific to tlscfg module.  Which is true for now. so shall i go along with the changes? )     

Thanks.

Reply:
I think `./pkg/config/tlscfg/testdata` is the ideal location, since it's the shared TLS-related code. Do you want to consolidate pem files in a different PR first and then come back to this one?

Reply:
Regarding your list, there are only two locations I see there, `./pkg/config/tlscfg/testdata` and `./cmd/agent/app/reporter/grpc/testdata`. The UI paths are external dependencies.

Reply:
Okay,  Thanks.  Ill make the changes and create another pull request.  Once that is merged,  ill delete thefiles in the query directory. 

Also, I am facing https://github.com/jaegertracing/jaeger/issues/2329 issue during CI build.  I have only made changes to the query package,  This shouldn't have happened.  Any workarounds / details that i missed?  Thanks,

Reply:
simply re-running the build usually helps, but we should really fix that one.

Reply:
Is there anyway to trigger rerunning the build without a new push?  Sorry.  I am new to CI as well.  Is an empty commit i.e.  `git commit --allow-empty` the way to go?

Reply:
Closing and opening PR works too.  Thanks.

Reply:
You can try `git commit --amend`, it should generate a new hash.